documentation/src/pages/recipes/data/recipes/smart-task-organizer.yaml

```diff
@@ -0,0 +1,115 @@
+version: 1.0.0
+title: Smart Task Organizer
+author:
+  contact: research@aegntic.ai
+description: Automatically organize and prioritize tasks from files, emails, and messages into an actionable todo list
+instructions: |
+  You are an intelligent task organizer that helps users turn scattered information into organized, actionable task lists. Your job is to scan various sources (files, messages, notes) and extract, categorize, and prioritize tasks effectively.
+
+  Focus on:
+  - Identifying concrete action items from text
+  - Categorizing tasks by urgency and importance
+  - Organizing tasks by project and context
+  - Providing clear next steps for each task
+activities:
+  - Scan and parse text files for action items
+  - Extract tasks from emails and messages
+  - Categorize tasks by priority and project
+  - Generate organized todo lists with deadlines
+  - Create follow-up reminders
+parameters:
+  - key: source_type
+    input_type: string
+    requirement: required
+    description: "Type of source to scan: files, emails, messages, notes, all"
+  - key: priority_level
+    input_type: string
+    requirement: optional
+    description: "Filter by priority: urgent, high, medium, low, all"
+    default: all
+  - key: project_filter
+    input_type: string
+    requirement: optional
+    description: "Filter tasks by specific project name"
+    default: ""
+extensions:
+  - type: builtin
```

Comment:
One small comment and then i can approve..filesystem is not a built in or bundled extension.

Amazing job and thanks for your contribution!